src/foam/u2/view/RadioButton.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'RadioButton',
+  extends: 'foam.u2.Element',
+
+  documentation: 'A single radio button. Logic implemented in RadioView.js',
+
+  imports: ['theme?'],
+
+  css: `
+  ^innerCircle.selected{
+    r: 5px;
+  }
+  ^innerCircle{
+    transition: 0.1s ease;
+    transform-origin: center center;
+    transform-box: stroke-box;
+    r: 0px;
+  }
+  `,
+
+  properties: [
+    {
+      name: 'selectedColor',
+      expression: function (isSelected, isDisabled) {
+        if ( isDisabled ) {
+          return this.theme.grey4;
```

Comment:
Theme is an optional import (because of the '?') so this should be made to work even if theme is missing. The theme will be missing for apps that use FOAM but not NANOS.

Reply:
This view is used in allViews as well so not using theme optional would break that. I'll add theme optional to RadioView as well